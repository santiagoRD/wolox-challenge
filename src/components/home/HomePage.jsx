import React, { lazy, Suspense } from 'react';
import Loader from '../../helpers/common/Loader';
import AboutSection from './sections/AboutSection';

const FollowSection = lazy(() => import('./sections/FollowSection'));
const BenefitsSection = lazy(() => import('./sections/BenefitsSection'));
const WelcomeSection = lazy(() => import('./sections/WelcomeSection'));

const HomePage = () => (
  <Suspense fallback={<Loader />}>
    <WelcomeSection />
    <FollowSection />
    <BenefitsSection />
    <AboutSection />
  </Suspense>
);

export default HomePage;
