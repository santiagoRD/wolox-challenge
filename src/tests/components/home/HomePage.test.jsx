import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../../components/home/HomePage';

describe('Testing <HomePage />', () => {
  test('should render <HomePage />', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
