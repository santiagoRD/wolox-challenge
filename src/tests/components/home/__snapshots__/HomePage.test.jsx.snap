// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HomePage /> should render <HomePage /> 1`] = `
<Suspense
  fallback={<Loader />}
>
  <lazy />
  <lazy />
  <lazy />
  <AboutSection />
</Suspense>
`;
